src/SerialTransceiver.h

```diff
@@ -1,3 +1,7 @@
+#include <string>
+// This is needed to compile std::string on esp8266.
+template class std::basic_string<char>;
```

Comment:
Shoudln't we need only that 1 time in a .cpp of the library? (IIRC it was a linker error)